project/plugins.sbt

```diff
@@ -10,3 +10,5 @@ addSbtPlugin(dependency = "org.scalariform" % "sbt-scalariform" % "1.8.3")
 addSbtPlugin(dependency = "org.scoverage" % "sbt-scoverage" % "2.0.7")
 addSbtPlugin(dependency = "org.scoverage" % "sbt-coveralls" % "1.3.11")
 addSbtPlugin(dependency = "org.xerial.sbt" % "sbt-sonatype" % "3.10.0")
+
+ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
```

Comment:
@ndeverge The problem here is that sbt-scapegoat pulls in scala-xml verson 1, but every other library already upgraded to scala-xml version 2. Given that scapegoat also does not come with Scala 3 artifacts yet (see https://github.com/scapegoat-scala/scapegoat/issues/521), I think we should just drop it... We don't use it in any other Play repos anyway.

Reply:
Also then this workaround is not necessary anymore.

Reply:
@mkurz Right, let's get rid of it then!